admin/pages/players.php

```diff
@@ -895,3 +895,22 @@ function updateOutfit()
             console.log(new_outfit);
         }
 	</script>
+<?php 
+// List all players
+$allAccounts = $db->query("SELECT id, name, account_id FROM players ORDER BY id ASC")->fetchAll();
```

Comment:
you are using the wrong variable name. It may cause time out with many values.

Reply:
Oh yes, I forgot to adapt this code for player variables, I'll adjust it here.